bucketeer/src/main/kotlin/io/bucketeer/sdk/android/internal/event/EventCreators.kt

```diff
@@ -84,6 +98,7 @@ internal fun newGoalEvent(
       tag = featureTag,
       source_id = SourceID.ANDROID,
       sdk_version = BuildConfig.SDK_VERSION,
+      metadata = newMetadata(appVersion),
```

Comment:
I just noticed that I forgot to add the `metadata` also to MetricsEvent proto message. I will add it on Monday, but you can add it in this PR if you like and merge it later.

Reply:
@yshrsmz 
I have added it to this PR.
https://github.com/bucketeer-io/bucketeer/pull/132

Reply:
I've added metadata to MetricsEvent in [`6aa09a9` (#35)](https://github.com/bucketeer-io/android-client-sdk/pull/35/commits/6aa09a94838705df6405f4de207925eff9b67472)